app/views/macros/recommend-form.njk

```diff
@@ -5,7 +5,7 @@
       <div class="govuk-panel govuk-panel--confirmation govuk-panel--align-horizontal govuk-!-text-align-center">
         <div class="govuk-panel__body">
           <div class="govuk-grid-row">
-            <button class="govuk-button" onclick="location.href='';">Recommend to pay</button>
```

Comment:
Don't use js, you can use Gov Button and other Gov Frontend style. 
https://design-system.service.gov.uk/components/